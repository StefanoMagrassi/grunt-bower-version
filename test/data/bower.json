{
  "name"       : "test-bower",
  "version"    : "0.0.0",
  "description": "Test bower.json",
  "private"    : true
}
