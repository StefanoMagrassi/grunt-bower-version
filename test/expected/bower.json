{
  "name"       : "test-bower",
  "version"    : "1.2.3",
  "description": "Test bower.json",
  "private"    : true
}
